src/core/server/ui_settings/settings/index.ts

```diff
@@ -15,14 +15,20 @@ import { getNotificationsSettings } from './notifications';
 import { getThemeSettings } from './theme';
 import { getStateSettings } from './state';
 
-export const getCoreSettings = (): Record<string, UiSettingsParams> => {
+interface GetCoreSettingsOptions {
+  isDist?: boolean;
+}
```

Comment:
NIT: why optional?

Reply:
Mostly because this is a temporary thing that I plan to remove once we not longer need to switch back to the v7 theme, so I wanted to avoid updating all the tests which use the `getCoreSettings()` fn.

Reply:
And if you're mostly curious about the property specifically, I didn't really want to push people to define the property if they somehow end up calling `getCoreSettings()` as I'm not really sure where it's called but the places where it matters the right options are passed.